mix.exs

```diff
@@ -30,19 +31,19 @@ defmodule App.Mixfile do
   #
   # Type `mix help deps` for examples and options.
   defp deps do
-    [{:phoenix, "~> 1.2.1"},
+    [{:phoenix, "~> 1.2"},
      {:phoenix_pubsub, "~> 1.0"},
      {:phoenix_ecto, "~> 3.0"},
      {:postgrex, ">= 0.0.0"},
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
      {:cowboy, "~> 1.0"},
-     {:absinthe, "~> 1.2.0"},
-     {:absinthe_plug, "~> 1.1"},
-     {:absinthe_ecto, git: "https://github.com/absinthe-graphql/absinthe_ecto.git"},
```

Comment:
A bit picky maybe but this follows master branch right? I've had issues with this before when the master branch changes and breaks my project even though seemingly nothing has changed. It's more stable to either follow a specific version or commit, is this possible? :)